File: app/src/main/java/com/rkbapps/neetflix/activityes/ImagePreviewActivity.java
package com.rkbapps.neetflix.activityes;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.MenuItem;
import android.widget.ImageView;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.bumptech.glide.Glide;
import com.google.android.material.appbar.MaterialToolbar;
import com.rkbapps.neetflix.R;

import java.util.Objects;

public class ImagePreviewActivity extends AppCompatActivity {

    @SuppressLint("MissingInflatedId")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_image_preview);
        ImageView preview = findViewById(R.id.imgPreview);
        MaterialToolbar toolbar = findViewById(R.id.toolbarImagePreview);

        setSupportActionBar(toolbar);
        Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setTitle("");

        String imageKey = getIntent().getStringExtra("imageKey");

        if (imageKey != null) {
            Glide.with(getApplicationContext()).load("https://image.tmdb.org/t/p/original/" + imageKey).into(preview);
        } else {
            Toast.makeText(this, "Something went wrong", Toast.LENGTH_SHORT).show();
        }
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        onBackPressed();
        return true;
    }
}

File: app/src/main/java/com/rkbapps/neetflix/activityes/ImagePreviewActivity.kt
package com.rkbapps.neetflix.activityes

import android.os.Bundle
import android.view.MenuItem
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import com.bumptech.glide.Glide
import com.rkbapps.neetflix.R
import com.rkbapps.neetflix.databinding.ActivityImagePreviewBinding

class ImagePreviewActivity : AppCompatActivity() {

    lateinit var binding: ActivityImagePreviewBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = DataBindingUtil.setContentView(this, R.layout.activity_image_preview)

        setSupportActionBar(binding.toolbarImagePreview)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.title = ""

        val imageKey = intent.getStringExtra("imageKey")
        if (imageKey != null) {
            Glide.with(this).load("https://image.tmdb.org/t/p/original/$imageKey")
                .into(binding.imgPreview)
        } else {
            Toast.makeText(this, "Something went wrong", Toast.LENGTH_SHORT).show()
        }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        onBackPressedDispatcher.onBackPressed()
        return true
    }
}